tests/JustSaying.UnitTests/JustSaying.UnitTests.csproj

```diff
@@ -1,6 +1,6 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <NoWarn>$(NoWarn);CA1034;CA1051;CA1707;CA2000;CA2007</NoWarn>
+    <NoWarn>$(NoWarn);CA1034;CA1051;CA1707;CA2000;CA2007;CA1054;CA2016</NoWarn>
```

Comment:
Better to not suppress `CA2016` and specify `CancellationToken.None` explicitly.

Reply:
Cool, will do 👍